src/IB/IBFEMethod.cpp

```diff
@@ -3238,17 +3037,38 @@ IBFEMethod::getFromInput(const Pointer<Database>& db, bool /*is_from_restart*/)
     else if (db->isBool("split_forces"))
         d_split_tangential_force = db->getBool("split_forces");
     if (db->isBool("use_jump_conditions")) d_use_jump_conditions = db->getBool("use_jump_conditions");
-    if (db->isBool("use_consistent_mass_matrix"))
-        d_use_consistent_mass_matrix = db->getBool("use_consistent_mass_matrix");
 
-    // Restart settings.
-    if (db->isString("libmesh_restart_file_extension"))
+    // FEDataManager settings.
+    if (db->keyExists("FEDataManager"))
+    {
+        d_fe_data_manager_db = db->getDatabase("FEDataManager");
+    }
+    else
+    {
+        d_fe_data_manager_db = new InputDatabase("FEDataManager");
+    }
+
+    // FEProjector settings.
```

Comment:
`d_fe_projector_db` is also set up by the base class. We should (until we completely remove it after the release) put the `FEProjector` database inside the `FEDataManager` database should it not be there already.

I recommend we replace this with
```cpp
if (!d_fe_data_manager_db->keyExists("FEProjector"))
{
    d_fe_data_manager_db->putDatabase(d_fe_projector_db);
}
```
and make it clear in the documentation that `FEProjector` should be given as a database inside the one passed to `IBFEMethod`/`FEMechanicsBase` et al and not inside the `FEDataManager` database.